client/src/component/PullRequestCard.js

```diff
@@ -14,12 +14,12 @@ const PullRequestCard = () => {
 
   return (
     <div>
-      <h2>Pull Requests</h2>
+      <h4>Pull Requests</h2>
```

Comment:
Opening and closing tags must match.

Headings shouldnt be used for styling, they must be in a tree structure across the document so if you have a h4 hear it must sit within a h3 parent. I know this is being loaded into another component but I dont think there is a h1, h2 and h3 above it